libs/wire-api/test/golden/Test/Wire/API/Golden/Generated/NewUserPublic_user.hs

```diff
@@ -38,7 +38,7 @@ testObject_NewUserPublic_user_1 =
         { newUserDisplayName =
             Name {fromName = "\\sY4]u\1033976\DLE\1027259\FS\ETX \US\ETB\1066640dw;}\1073386@\184511\r8"},
           newUserUUID = Nothing,
-          newUserIdentity = Just (EmailIdentity (Email {emailLocal = "test", emailDomain = "example.com"})),
+          newUserIdentity = Just (EmailIdentity (unsafeEmailAddress "some" "example")),
```

Comment:
Here, I see no value in changing the value. Have you changed this in an automated fashion, hence the change?

Reply:
Correct :)